### DISCUSSION TITLE
Webhooks: Duplicate Requests?

### DISCUSSION THREAD

**Discussion by sebastiansmithomm** [directus/directus#13925] [Archive]
Hello! Just a quick question on webhooks, I have set up a new installation of directus (docker) with a single table, and a new webhook with 'create' trigger. The hook calls a node endpoint on port 4000.

The webhook is always triggered twice in quick succession. I have confirmed this by using Fiddler (see below). 

![image](https://user-images.githubusercontent.com/74179584/174024390-5b0c408e-fff2-40c4-a58e-74e9086c769c.png)

Here is my webhook setup: 
![image](https://user-images.githubusercontent.com/74179584/174024637-98baf9e5-62a6-44e4-84d3-9f268a71e379.png)

Is this behaviour expected? Or is it a misconfiguration on my part?

**Comment by br41nslug** [comment]
could one be an `OPTIONS` request followed by the `GET` request?

**Reply by sebastiansmithomm** [reply]
Good idea, but both are GET
![image](https://user-images.githubusercontent.com/74179584/174026406-01669599-00cb-4f27-b519-693e31bae92e.png)

**Reply by Jeyakumarrathnaselvi** [reply]
It is happening in latest version
